src/nrtk_explorer/app/ui/image_list.py

```diff
@@ -316,10 +350,10 @@ def __init__(self, on_scroll, on_hover, **kwargs):
                                         "Transformed: Detection Annotations",
                                         classes="text-center",
                                     )
-                                    ImageDetection(
+                                    ImageWithSpinner(
                                         identifier=("props.row.transformed",),
-                                        src=("get(props.row.transformed)",),
-                                        annotations=("props.row.transformedAnnotations",),
+                                        src="get(props.row.transformed)",
```

Comment:
`src=("get(props.row.transformed)",),`

Reply:
It was a string so I could check the value of transformed in the show spinner logic.  Better way?
```
            quasar.QInnerLoading(
                showing=(
                    # transformed image src props are refs, so check if ref is not set
                    f"(typeof {src} === 'object' && !{src}.value) || (show_annotations_on_images && !{annotations}.value)",
                )
            )
```

Reply:
the get() method return a ref. Editing showing is not a good option.

Reply:
Shucks, I'm still not seeing how to write the `showing` expression.  I chaged it to: 
```
showing=(f"!{src} || (show_annotations_on_images && !{annotations}.value)",)`, 
```
but get JS syntax error:
```
{ showing: !('get(props.row.transformed_src).value',) || (show_annotations_on_images && !props.row.transformedAnnotations.value) }
```

Reply:
just make your src expect a tuple and just resolve it in showing=f"{src[0]} ..."

Reply:
Ah, naturaly, thank you.